A0/cdhuyen-HelloWorld.py

```diff
@@ -0,0 +1,11 @@
+import argparse
```

Comment:
please add the she-bang line
```suggestion
#!/usr/bin/env python3
import argparse
```